docs/_build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
```

Comment:
I don't really know anything about Sphinx. Is this file meant to be tracked by git?